Modules/IO/MeshBase/include/itkMeshFileReader.h

```diff
@@ -187,6 +187,29 @@ class ITK_TEMPLATE_EXPORT MeshFileReader : public MeshSource<TOutputMesh>
 private:
   std::string m_ExceptionMessage;
 };
+
+
+/** Convenience function for reading a mesh.
+ *
+ * `TOutputMesh` is the expected output mesh type, and the optional
+ * `ConvertPointPixelTraits`, ``ConvertCellPixelTraits` template parameters are used to do the conversion,
+ * as specified by MeshFileReader.
+ *
+ * The function reads the mesh from the specified file, and returns the
+ * mesh that it has read.
+ * */
+template <typename TOutputMesh,
+          typename ConvertPointPixelTraits = MeshConvertPixelTraits<typename TOutputMesh::PixelType>,
+          class ConvertCellPixelTraits = MeshConvertPixelTraits<typename TOutputMesh::CellPixelType>>
```

Comment:
Why call one a `typename` and the other a class?

Reply:
This is just a result of copy-paste. I will use `typename` in both places.